aries_cloudagent/indy/models/cred_abstract.py

```diff
@@ -331,22 +331,22 @@ class Meta:
         model_class = VCDICredAbstract
         unknown = EXCLUDE
 
-        data_model_versions_supported = fields.List(
-            fields.Str(), required=True, metadata={"description": "", "example": ""}
-        )
```

Comment:
@tra371 @supersonicwisd1 so this means, these are not supposed to be metadata, they are supposed to be in the credential?